content/zh/docs/tasks/observability/metrics/telemetry-api/index.md

```diff
@@ -59,9 +59,6 @@ spec:
 
 1. 为 `REQUEST_COUNT` 指标添加自定义标签
 
-    Telemetry API 无法更新 `MeshConfig` 中的 `extraStatTags`，
```

Comment:
Chinese version can be handled by @asa3311 in #12958

Reply:
that's tracing